includes/helper/class-wp-job-manager-helper.php

```diff
@@ -594,7 +655,7 @@ private function handle_api_errors( $product_slug, $response ) {
 		}
 
 		$errors         = ! empty( $response['errors'] ) ? $response['errors'] : [];
-		$allowed_errors = [ 'no_activation', 'expired_key', 'expiring_soon' ];
+		$allowed_errors = [ 'no_activation', 'expired_key', 'expiring_soon', 'update_available' ];
```

Comment:
This will allow us to notify the users if they don't have a license but there is a pending update available. For now, it will just show in the license area, but eventually we could use this to hint on the plugins screen.

Note: This isn't implemented on WPJobManager.com.